.circleci/config.yml

```diff
@@ -3,6 +3,8 @@
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
 #
 version: 2
+orbs:
+  queue: eddiewebb/queue@volatile
```

Comment:
the `volatile` version is a bit scary. can we use the latest version [1.1.2](https://circleci.com/orbs/registry/orb/eddiewebb/queue)?

Reply:
:+1: updated to use 1.1.2